qutip/core/gates.py

```diff
@@ -738,10 +815,14 @@ def qubit_clifford_group(*, dtype="dense"):
     # product(...) yields the Cartesian product of its arguments.
     # Here, each element is a tuple (E**i, X**j, S**k) such that
     # partial(reduce, mul) acting on the tuple yields E**i * X**j * S**k.
-    return [
+    gates = [
         op.to(dtype)
         for op in map(
             partial(reduce, mul),
             product(_powers(E, 3), _powers(X, 2), _powers(S, 4)),
         )
     ]
+    for gate in gates:
+        gate.isherm
```

Comment:
Set the flags, since there are 24 of them with some being hermitian other not, I call the method compute them. Since they are all 2x2, it does not that a significant time.